### PR TITLE
[DRAFT] Folds Bog and Watchmen together

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request

This is going to be controversial for a lot of reasons but the main issue is that Watchmen and Bog are redundant. To make a comparison, its like Dept Sec on Skyrat servers -- they effectively become their own antagonist in the worse of cases and in the best of cases they don't exist. Departmental Security also encourages factionalism with departments when they are supposed to be working towards one goal, defeating the antagonist. Before accusations are made, I used to be a Sheriff main.

Anyway, this PR ensures the new city watch can't be inhumane worshippers and fulfill the role as security for the town of Rockhill. Regular watchmen get a listening stone, the Sheriff gets a ring. Also to reflect their status, their mister account is no longer dirt poor (lol?)

## Why It's Good For The Game

Mentioned above. Discuss.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/garrison/townguard.dm [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 	allowed_races = RACES_ALL_KINDS
+	allowed_patrons = ALL_DIVINE_PATRONS // they should not be antags.
 	allowed_ages = list(AGE_ADULT, AGE_MIDDLEAGED)
-	tutorial = "Responsible for the safety of the town and the enforcement of the King's law, you are the vanguard of the city faced with punishing those who defy his Royal Majesty. Though you've many lords to obey, as both the Church and the Judge have great sway over your life."
-	display_order = JDO_TOWNGUARD
+	tutorial = tutorial = "You are a member of the now consolidated city watch after the merger with the Bogmen. As such, you have been granted more training and gone through basic purity exercises as well. You are the last stand against whatever is out there in the bog and the town itself. Good luck."
 	whitelist_req = TRUE
 
 	outfit = /datum/outfit/job/roguetown/guardsman
+		allowed_patrons = ALL_DIVINE_PATRONS
```
Comment (Evie7056) [inline code/modules/jobs/job_types/roguetown/garrison/townguard.dm:17]: extra tab

File: code/modules/jobs/job_types/roguetown/garrison/sheriff.dm [hunk 2]
```diff
@@ -6,13 +6,15 @@
...
+	allowed_patrons = ALL_DIVINE_PATRONS
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_ages = list(AGE_ADULT, AGE_MIDDLEAGED, AGE_OLD)
 	display_order = JDO_SHERIFF
-	tutorial = "Crime has always been a constant of your life, and you always chose the side of justice. You rose up through the ranks as a watchman, and now rule over them - Ensure that they enforce the laws of this land properly."
+	tutorial = "Granted nobility from the ruling lord of the land, you manage the newly consolidated watch. With the brawn of the bogmen and skill of the watch proper you maintain an indomitable force of men that will slay any creature that gets past the walls of the bog gate."
 	whitelist_req = FALSE
 	outfit = /datum/outfit/job/roguetown/sheriff
-	give_bank_account = 26
+		allowed_patrons = ALL_DIVINE_PATRONS
```
Comment (Evie7056) [inline code/modules/jobs/job_types/roguetown/garrison/sheriff.dm:16]: extra tab

File: code/modules/jobs/job_types/roguetown/garrison/townguard.dm [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
+	total_positions = 5
+	spawn_positions = 5 // recruit more IC!
 	selection_color = JCOLOR_SOLDIER
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_ALL_KINDS
+	allowed_patrons = ALL_DIVINE_PATRONS // they should not be antags.
 	allowed_ages = list(AGE_ADULT, AGE_MIDDLEAGED)
-	tutorial = "Responsible for the safety of the town and the enforcement of the King's law, you are the vanguard of the city faced with punishing those who defy his Royal Majesty. Though you've many lords to obey, as both the Church and the Judge have great sway over your life."
-	display_order = JDO_TOWNGUARD
+	tutorial = tutorial = "You are a member of the now consolidated city watch after the merger with the Bogmen. As such, you have been granted more training and gone through basic purity exercises as well. You are the last stand against whatever is out there in the bog and the town itself. Good luck."
```
Comment (Evie7056) [inline code/modules/jobs/job_types/roguetown/garrison/townguard.dm:13]: `tutorial = tutorial`
wont compile for sure. remove a var inside a var

File: code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm [hunk 4]
```diff
@@ -15,6 +15,7 @@
 
 	spells = list(/obj/effect/proc_holder/spell/self/convertrole/bog)
 	outfit = /datum/outfit/job/roguetown/bogmaster
+		allowed_patrons = ALL_DIVINE_PATRONS
```
Comment (Evie7056) [inline code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm:18]: extra tab

Comment (Evie7056) [reply code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm:18]: may not necessary cause compile error, but it it's still not good to do this

**Review by Evie7056 — CHANGES_REQUESTED** [review]
Please look onto the compile error. This also has few styling errors that may or may not cause compile fail.

File: code/modules/jobs/job_types/roguetown/garrison/townguard.dm [hunk 5]
```diff
@@ -1,22 +1,23 @@
...
-	spawn_positions = 8
+	total_positions = 5
+	spawn_positions = 5 // recruit more IC!
 	selection_color = JCOLOR_SOLDIER
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_ALL_KINDS
+	allowed_patrons = ALL_DIVINE_PATRONS // they should not be antags.
 	allowed_ages = list(AGE_ADULT, AGE_MIDDLEAGED)
-	tutorial = "Responsible for the safety of the town and the enforcement of the King's law, you are the vanguard of the city faced with punishing those who defy his Royal Majesty. Though you've many lords to obey, as both the Church and the Judge have great sway over your life."
-	display_order = JDO_TOWNGUARD
```
Comment (Evie7056) [inline code/modules/jobs/job_types/roguetown/garrison/townguard.dm:13]: I think you also ate here an extra line

**Comment by oprimeolt** [comment]
I'd say keep bogmaster as just 'mans the bog gate'

**Comment by Feralape** [comment]
Won't switching total positions to 0 make it an infinite slot?

**Comment by Decoy1240** [comment]
> Won't switching total positions to 0 make it an infinite slot?

No -1 I think is how you do that look at begger?

**Comment by Decoy1240** [comment]
> I'd say keep bogmaster as just 'mans the bog gate' 

This probably would be merged with a update to also remove the bog gate and add a new one

**Comment by AllCorruptingWyrm** [comment]
due to a vote difference of 3, a compromise will be met.

**Pull request closed without merging**